update(UI overview): add bonita apps (#1362)
bonitasoft/bonita-doc: md/admin-application-roles-list.md

# Role list in the Bonita Administrator Application

> This page explains what a user with the _Administrator_ profile in Bonita Administrator Application can see and do about roles in the organization.  

Reviewer: ```suggestion
This page explains what a user with the _Administrator_ profile can see and do about roles (of the organization) in [Bonita Administrator Application](admin-application-overview).  
```

> To easily manipulate users with the same rights, it is possible to add them in the same role, like "Project Managers", for example.   

Reviewer: ```suggestion
To easily manipulate users with the same rights, it is possible to give them the same role (in the same or in different groups), like "Project Manager", for example.   
```

> Roles can be defined in the organization, users can then be added to roles, and roles are used to map actors or profiles or other permissions in Bonita.

Reviewer: ```suggestion
Roles can be defined in the organization, users can then be given the roles (within a group), and roles can be used to map actors or profiles or other permissions in Bonita.
```

Author: roles can be given even if the user belongs to no group yet. I'm keeping the "can be" though.

Reviewer: yes.
What I was trying to say is that it not possible to give a role to a user without adding him in a group at the same time or changing an existing membership. but you already explaining the membership concept the bottom of the page.

## Live Update
The Administrator can [Live Update](live-update.md) a role:
 * create
 * delete
 * edit the name and description
 * modify users within
This is only in case of an issue to solve temporarily. We strongly encourage the modification of the organization to be conducted 
in Bonita Studio so it can be shared and versioned with the whole project, or through the [LDAP synchronizer](ldap-synchronizer.md) if the organization is located in an external system. 

### Create a role
1. Go to _Organization_/_Roles_
2. Click on the _Create_ button
3. In the modal window, enter the name (can be technical), the display name, and a description of the role
4. Click on _Create_
The new role will be displayed in the list of roles.

### Delete a role
1. Go to _Organization_/_Roles_
2. In the row of the role, click on the _Delete_ icon
3. In the modal window, click on _Delete_
4. In the confirmation modal, after checking that it is safe to delete, click on _Delete_

### Edit a role name and description
1. Go to _Organization_/_Roles_
2. On the row of the role, click on the _Edit role details_ icon
3. In the modal window, update the role information
4. Click _Save_

## Modify users in a role

> In the way we conceive the pertainance of a user to an organization, it is only possible to add both a role and a group (what we call a membership) to a user.  

Reviewer: pertainance ?
```suggestion
In the way we conceive the belonging of a user to an organization, it is only possible to add both a role and a group (what we call a membership) to a user.  
```

Author: same as for group. Have been too creative here
